### PR TITLE
Data driven circle color example

### PR DESCRIPTION
**Pull request by tmcw**
Just puts @peterqliu's example in #2439 into an example.

cc @lucaswoj for the review

### CODE REVIEW

**Comment by peterqliu** [comment]
Maybe a legend?

**Comment by tmcw** [comment]
I think we should keep this barebones for now and not have a legend. We can demo legends in other examples.

File: docs/_posts/examples/3400-01-08-data-driven-circle-colors.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+layout: example
+category: example
+title: Data-driven circle coloring by ethnicity
+description: Using a categorical circle-color property for a visualization
+---
+<div id='map'></div>
+<script>
+var map = new mapboxgl.Map({
+  container: 'map',
+  style: 'mapbox://styles/mapbox/light-v8',
```
Comment (lucaswoj) [inline docs/_posts/examples/3400-01-08-data-driven-circle-colors.html]: Most examples use 4 space tabbing. ref https://github.com/mapbox/mapbox-gl-js/issues/1781

**Comment by lucaswoj** [comment]
LGTM, 🚢  once you've fixed the formatting!

**Comment by tmcw** [comment]
👍 now 4 space indent

**Pull request merged**